content/en/bits_ai/query_examples.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Example Natural Language Queries
+kind: guide
+disable_toc: false
+private: true
+is_beta: true
+---
+
+{{< whatsnext desc="For instructions on running natural language queries in Datadog, see:">}}
+    {{< nextlink href="bits_ai/getting_started/#in-the-chat-panel" >}}Querying in the chat panel{{< /nextlink >}}
+    {{< nextlink href="bits_ai/getting_started/#in-a-search-bar" >}}Querying in a search bar{{< /nextlink >}}
+{{< /whatsnext >}}
+
+## Logs
+You can query your logs in natural language from the Logs Explorer search bar.
+
+- "Find errors in AWS CloudTrail where a user is assuming a different user's role"
+{{< img src="bits_ai/cloudtrail-user-role-errors.png" alt="Screenshot of query result" style="width:90%;">}}
+- "Create a pie chart of error logs by service"
+{{< img src="bits_ai/logs-pie-chart.png" alt="Screenshot of query result" style="width:90%;">}}
+- "Show me patterns of errors for users checking out"
+{{< img src="bits_ai/checkout-error-patterns.png" alt="Screenshot of query result" style="width:90%;">}}
+
+## APM traces
+
+To query APM traces, use the chat panel.
+
+Examples:
+- "Show me traces for web-store that are slower than 1s"
+{{< img src="bits_ai/slow-web-store-traces.png" alt="Screenshot of query result" style="width:90%;">}}
+
+## Infrastructure data (Inventories SQL)
+
+Infrastructure resource data can be queried in natural language from the search bar of [Inventories SQL][3], a centralized interface for infrastructure resource data. You can also use the chat panel to query infrastructure data from anywhere in Datadog.
+
+Examples:
+- "How many hosts am I running by availability zone?"
+{{< img src="bits_ai/hosts-by-az.png" alt="Screenshot of query result" style="width:90%;">}}
+- "Show me the security groups open to the Internet"
+{{< img src="bits_ai/open-security-groups.png" alt="Screenshot of query result" style="width:90%;">}}
+
+## Cloud cost
+
+Natural language querying of cloud cost is available through the in-product search bar or the chat panel.
+
+Examples:
+- "How much does the payments team spend?"
```

Comment:
So the queries didn't actually translate correctly in the first two examples. Could you replace the three examples you currently have with these two: 

"Show me how much each team spends on the web-store service" 
https://app.datadoghq.com/cost/analytics?query=sum%3Aaws.cost.net.amortized.cpu.allocated%7Bservice%3Aweb-store%7D%20by%20%7Bteam%7D.rollup%28sum%2C%2086400%29&anomaliesOnly=false&start=1679529600000&end=1695495778152&paused=false

"AWS products with >5% increase in costs"
https://app.datadoghq.com/cost/analytics?query=sum%3Aaws.cost.net.amortized.cpu.allocated%7B%2A%7D%20by%20%7Baws_product%7D.rollup%28sum%2C%2086400%29&anomaliesOnly=false&start=1679529600000&end=1695495778152&paused=false